docs/source/config_file.rst

```diff
@@ -714,6 +714,13 @@ section of the command line docs.
    Prohibit equality checks, identity checks, and container checks between
    non-overlapping types.
 
+.. confval:: strict_override_decorator
+
+    :type: boolean
+    :default: False
+
+    Require ``override`` decorator if method is overriding a base class method.
```

Comment:
Maybe write "Require `@override` decorator ..." to make things a bit more explicit, since override is a common word.